### ISSUE TITLE
Doesn't work from REPL

### ISSUE THREAD

**Issue by lewisl** [JuliaComputing/TableView.jl#88]
```
julia> using TypedTables

julia> using Blink

julia> using TableView

julia> tt = Table(a=rand(100), b=zeros(Int,100));

julia> TableView.showtable(tt)
WebIO.Scope(WebIO.Node{WebIO.DOM}(WebIO.DOM(:html, :div), Any[WebIO.Node{WebIO.DOM}(WebIO.DOM(:html, :div), Any[""], Dict{Symbol, Any}(:style => Dict("height" => "0", "padding" => "0", "background-color" => "#F5F7F7", "color" => "#1c1f20", "font-family" => "-apple-system, BlinkMacSystemFont, \"Segoe UI\", Roboto, Oxygen-Sans, Ubuntu, Cantarell, \"Helvetica Neue\", sans-serif"))), WebIO.Node{WebIO.DOM}(WebIO.DOM(:html, :div), Any[], Dict{Symbol, Any}(:id => "grid-9f1bd4d4", :className => "ag-theme-balham", :style => Dict("height" => "500px", "width" => "100%")))], Dict{Symbol, Any}()), Dict{String, Tuple{Observables.AbstractObservable, Union{Nothing, Bool}}}("onCellKeyDown" => (Observable{Any} with 0 listeners. Value:
nothing, nothing), "onRowClicked" => (Observable{Any} with 0 listeners. Value:
nothing, nothing), "onCellDoubleClicked" => (Observable{Any} with 0 listeners. Value:
nothing, nothing), "onCellClicked" => (Observable{Any} with 0 listeners. Value:
nothing, nothing), "onCellFocused" => (Observable{Any} with 0 listeners. Value:
nothing, nothing)), Set{String}(), nothing, WebIO.Asset[WebIO.Asset("js", nothing, "/Users/lewis/.julia/packages/TableView/CaHPi/src/rowNumberRenderer.js"), WebIO.Asset("js", nothing, "/Users/lewis/.julia/packages/TableView/CaHPi/deps/ag-grid-25.2.0/ag-grid.js"), WebIO.Asset("css", nothing, "/Users/lewis/.julia/packages/TableView/CaHPi/deps/ag-grid-25.2.0/ag-grid.css"), WebIO.Asset("css", nothing, "/Users/lewis/.julia/packages/TableView/CaHPi/deps/ag-grid-25.2.0/ag-grid-light.css"), WebIO.Asset("css", nothing, "/Users/lewis/.julia/packages/TableView/CaHPi/deps/ag-grid-25.2.0/ag-grid-dark.css")], Dict{Any, Any}(), WebIO.ConnectionPool(Channel{Any}(32), Set{WebIO.AbstractConnection}(), Condition(Base.InvasiveLinkedList{Task}(Task (runnable) @0x000000010b31e570, Task (runnable) @0x000000010b31e570), Base.AlwaysLockedST(1))), WebIO.JSString[WebIO.JSString("function () {\n    var handler = ((function (RowNumberRenderer,agGrid){var gridOptions={\"onCellFocused\":(function (ev){var x={}; (ev.rowIndex!==undefined) ? (x[\"rowIndex\"]=(ev.rowIndex+1)) : undefined; (ev.colDef!==undefined) ? (x[\"column\"]=ev.colDef.headerName) : undefined; return WebIO.setval({\"name\":\"onCellFocused\",\"scope\":\"872398187265855670\",\"id\":\"12188640123139181595\",\"type\":\"observable\"},x)}),\"onCellValueChanged\":(function (){return }),\"rowSelection\":\"multiple\",\"multiSortKey\":\"ctrl\",\"columnDefs\":[{\"filter\":false,\"editable\":false,\"headerTooltip\":\"\",\"field\":\"__row__\",\"headerName\":\"Row\",\"sortable\":true,\"type\":\"numericColumn\",\"cellRenderer\":\"rowNumberRenderer\",\"resizable\":true},{\"filter\":\"agNumberColumnFilter\",\"editable\":false,\"headerTooltip\":\"Float64\",\"field\":\"a\",\"headerName\":\"a\",\"sortable\":true,\"type\":\"numericColumn\",\"resizable\":true},{\"filter\":\"agNumberColumnFilter\",\"editable\":false,\"headerTooltip\":\"Int64\",\"field\":\"b\",\"headerName\":\"b\",\"sortable\":true,\"type\":\"numericColumn\",\"resizable\":true}],\"onCellDoubleClicked\":(function (ev){var x={}; (ev.rowIndex!==undefined) ? (x[\"rowIndex\"]=(ev.rowIndex+1)) : undefined; (ev.colDef!==undefined) ? (x[\"column\"]=ev.colDef.headerName) : undefined; return WebIO.setval({\"name\":\"onCellDoubleClicked\",\"scope\":\"872398187265855670\",\"id\":\"14024851735965581102\",\"type\":\"observable\"},x)}),\"onRowClicked\":(function (ev){var x={}; (ev.rowIndex!==undefined) ? (x[\"rowIndex\"]=(ev.rowIndex+1)) : undefined; (ev.colDef!==undefined) ? (x[\"column\"]=ev.colDef.headerName) : undefined; return WebIO.setval({\"name\":\"onRowClicked\",\"scope\":\"872398187265855670\",\"id\":\"8528506991506749490\",\"type\":\"observable\"},x)}),\"onCellKeyDown\":(function (ev){var x={}; (ev.rowIndex!==undefined) ? (x[\"rowIndex\"]=(ev.rowIndex+1)) : undefined; (ev.colDef!==undefined) ? (x[\"column\"]=ev.colDef.headerName) : undefined; return WebIO.setval({\"name\":\"onCellKeyDown\",\"scope\":\"872398187265855670\",\"id\":\"4650459912949644984\",\"type\":\"observable\"},x)}),\"rowData\":[{\"a\":0.7985391505141592,\"b\":0},{\"a\":0.14118612629674165,\"b\":0},{\"a\":0.8419876814117311,\"b\":0},{\"a\":0.5976947145844016,\"b\":0},{\"a\":0.9889480216177916,\"b\":0},{\"a\":0.05240395596165914,\"b\":0},{\"a\":0.7635614800089872,\"b\":0},{\"a\":0.9575489850037494,\"b\":0},{\"a\":0.9220815774206674,\"b\":0},{\"a\":0.212465897580152,\"b\":0},{\"a\":0.4093240277115765,\"b\":0},{\"a\":0.19927510731626696,\"b\":0},{\"a\":0.5305028956507498,\"b\":0},{\"a\":0.8711439269484351,\"b\":0},{\"a\":0.7415680471209041,\"b\":0},{\"a\":0.581094487774642,\"b\":0},{\"a\":0.11615044472785818,\"b\":0},{\"a\":0.3569789923507345,\"b\":0},{\"a\":0.3657513022519392,\"b\":0},{\"a\":0.10305068666926043,\"b\":0},{\"a\":0.9797755733515161,\"b\":0},{\"a\":0.9605357267965461,\"b\":0},{\"a\":0.1779448936785164,\"b\":0},{\"a\":0.6676134628619711,\"b\":0},{\"a\":0.7236933178098871,\"b\":0},{\"a\":0.18810388129768518,\"b\":0},{\"a\":0.5254968087201554,\"b\":0},{\"a\":0.8709848342135692,\"b\":0},{\"a\":0.536505503001479,\"b\":0},{\"a\":0.3813496542367436,\"b\":0},{\"a\":0.9475716411579345,\"b\":0},{\"a\":0.288624913975689,\"b\":0},{\"a\":0.25063590255496926,\"b\":0},{\"a\":0.944905749852068,\"b\":0},{\"a\":0.6464762174991082,\"b\":0},{\"a\":0.8957737602267535,\"b\":0},{\"a\":0.5753094952248194,\"b\":0},{\"a\":0.7992279836616962,\"b\":0},{\"a\":0.025575904092802304,\"b\":0},{\"a\":0.7078062352396021,\"b\":0},{\"a\":0.33699901433656565,\"b\":0},{\"a\":0.4164202097829769,\"b\":0},{\"a\":0.8905850083687606,\"b\":0},{\"a\":0.9719786846397032,\"b\":0},{\"a\":0.7667116319899396,\"b\":0},{\"a\":0.8751262088409707,\"b\":0},{\"a\":0.3805315428261681,\"b\":0},{\"a\":0.5238102729720111,\"b\":0},{\"a\":0.14691140614986586,\"b\":0},{\"a\":0.5889076694135252,\"b\":0},{\"a\":0.9393253716349008,\"b\":0},{\"a\":0.33302273502590773,\"b\":0},{\"a\":0.950407445114698,\"b\":0},{\"a\":0.3552871690021414,\"b\":0},{\"a\":0.6400872411137272,\"b\":0},{\"a\":0.2954559194312202,\"b\":0},{\"a\":0.9376290932266154,\"b\":0},{\"a\":0.8055889101933927,\"b\":0},{\"a\":0.8149537588568335,\"b\":0},{\"a\":0.33485842615892236,\"b\":0},{\"a\":0.7076654660685582,\"b\":0},{\"a\":0.7680466461126664,\"b\":0},{\"a\":0.21582028154362765,\"b\":0},{\"a\":0.1456724906850685,\"b\":0},{\"a\":0.126192650966774,\"b\":0},{\"a\":0.4536299537498728,\"b\":0},{\"a\":0.05383569649526776,\"b\":0},{\"a\":0.9587953902662429,\"b\":0},{\"a\":0.20850961920404032,\"b\":0},{\"a\":0.7474332655888565,\"b\":0},{\"a\":0.6286191878057118,\"b\":0},{\"a\":0.983719716453161,\"b\":0},{\"a\":0.9268411888281652,\"b\":0},{\"a\":0.32537597193679035,\"b\":0},{\"a\":0.5996169290930924,\"b\":0},{\"a\":0.41680566610782055,\"b\":0},{\"a\":0.7126009097045412,\"b\":0},{\"a\":0.7480303639690409,\"b\":0},{\"a\":0.794159678976464,\"b\":0},{\"a\":0.9426982171064615,\"b\":0},{\"a\":0.9373136041723295,\"b\":0},{\"a\":0.1556142702230825,\"b\":0},{\"a\":0.3774397394824065,\"b\":0},{\"a\":0.3423938449815991,\"b\":0},{\"a\":0.19477291623682524,\"b\":0},{\"a\":0.48690945921725526,\"b\":0},{\"a\":0.05684201703320457,\"b\":0},{\"a\":0.9646124479163009,\"b\":0},{\"a\":0.7972749930558484,\"b\":0},{\"a\":0.313504297750717,\"b\":0},{\"a\":0.8973924353658123,\"b\":0},{\"a\":0.9196401807107665,\"b\":0},{\"a\":0.7290882155795828,\"b\":0},{\"a\":0.08493849638931372,\"b\":0},{\"a\":0.27128817921247184,\"b\":0},{\"a\":0.7605819583385357,\"b\":0},{\"a\":0.08303632578249476,\"b\":0},{\"a\":0.9795826227575581,\"b\":0},{\"a\":0.5172542984020722,\"b\":0},{\"a\":0.03096640459020139,\"b\":0}],\"onCellClicked\":(function (ev){var x={}; (ev.rowIndex!==undefined) ? (x[\"rowIndex\"]=(ev.rowIndex+1)) : undefined; (ev.colDef!==undefined) ? (x[\"column\"]=ev.colDef.headerName) : undefined; return WebIO.setval({\"name\":\"onCellClicked\",\"scope\":\"872398187265855670\",\"id\":\"13732312895473466637\",\"type\":\"observable\"},x)})}; var el=document.getElementById(\"grid-9f1bd4d4\"); gridOptions.components={\"rowNumberRenderer\":RowNumberRenderer}; false ? (agGrid.LicenseManager.setLicenseKey(null)) : undefined; this.table=new agGrid.Grid(el,gridOptions); return gridOptions.columnApi.autoSizeAllColumns()}));\n    (WebIO.importBlock({\"data\":[{\"name\":null,\"type\":\"js\",\"url\":\"/assetserver/fda5317fc71dd60e4cb2eab183d7638278ee943e-rowNumberRenderer.js\"},{\"name\":null,\"type\":\"js\",\"url\":\"/assetserver/5871cc261db021f853f9731b943dbeaf79318c97-ag-grid.js\"},{\"name\":null,\"type\":\"css\",\"url\":\"/assetserver/113a1808386103d4f1ee22f86cffb58f76afe821-ag-grid.css\"},{\"name\":null,\"type\":\"css\",\"url\":\"/assetserver/e33b0a3839ed557493f4a29632a38e7b94b81d1d-ag-grid-light.css\"},{\"name\":null,\"type\":\"css\",\"url\":\"/assetserver/67e9ef70b26e13092a6a5fe767a760838cc3bdf4-ag-grid-dark.css\"}],\"type\":\"async_block\"})).then((imports) => handler.apply(this, imports));\n}\n")])
```

**Comment by pfitzseb** [comment]
Please check the screenshot in the readme for displaying any WebIO scope in a Blink window.

**Comment by lewisl** [comment]
Wasn’t sure if that was output or the commands to run.   So, a lot to run that ought to be part of the API of TableView.

Works well with Jupyter Lab.  I’ll use it in that context.

From: Sebastian Pfitzner ***@***.***>
Reply-To: "JuliaComputing/TableView.jl" ***@***.***>
Date: Friday, June 3, 2022 at 1:22 PM
To: "JuliaComputing/TableView.jl" ***@***.***>
Cc: Lewis Levin ***@***.***>, Author ***@***.***>
Subject: Re: [JuliaComputing/TableView.jl] Doesn't work from REPL (Issue #88)


Closed #88<https://github.com/JuliaComputing/TableView.jl/issues/88> as completed.
